### ISSUE TITLE
Inconsistency between versionFormat and valueFormat

### ISSUE THREAD

**Issue by prgp** [qoomon/maven-git-versioning-extension#91]
There is an inconsistency with the way in which `versionFormat` and `valueFormat` handle references.

For a branch with a pattern like `feature/TICKET/DESC` and `maven-git-versioning-extension.xml` containing:

```
<branch>
    <pattern><![CDATA[feature/(?<feature>.+)]]></pattern>
    <versionFormat>feature-${feature}-SNAPSHOT</versionFormat>
    <property>
        <pattern>service\.version</pattern>
        <valueFormat>feature-${feature}-SNAPSHOT</valueFormat>
    </property>
</branch>
```

We get:
```
[INFO] ----------------- maven-git-versioning-extension:5.3.0 -----------------
[INFO] Adjusting project models...
[INFO] 
[INFO] --- Grp:Aid @ branch feature/TICKET/DESC ---
[INFO] project version: feature-TICKET-DESC-SNAPSHOT
[INFO] properties:
[INFO]   service.version: feature-TICKET/DESC-SNAPSHOT
[INFO] 
```

The project version is exactly as desired, but somehow the property is not set to the same value even though it has the same format - this cause maven to fail because there is a `/` in the version.

It seems as if the "version" and "value" format code is slightly different WRT the generation of the value to be substituted.

I tried to use `${feature.slug}, but that absolutely did not give the desired result (FYI it gave `feature-${feature.slug}-SNAPSHOT`).

This issue existed in 5.2.x too

**Comment by qoomon** [comment]
Thanks for reporting. I will add the `.slug` option to all git placeholders. Currently this feature is only implemented for the `ref ` placeholder.

**Comment by prgp** [comment]
Being able to get the `slug` form for the capture groups would be awesome 👍

**Comment by qoomon** [comment]
I do a major refactoring currently so, it will probably takes a week until I implement this feature. However its a easy to implement feature.

**Comment by qoomon** [comment]
Just released with version `6.0.1`. Be aware of breaking changes see [CHANGELOG.md](https://github.com/qoomon/maven-git-versioning-extension/blob/master/CHANGELOG.md)
